guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -2,18 +2,25 @@
 = Installing PostgreSQL
 
 You can install only the same version of PostgreSQL that is installed with the `{foreman-installer}` tool during an internal database installation.
-{Project} supports PostgreSQL version 12.
+{Project} supports PostgreSQL version 13.
 
 include::snip_firewalld.adoc[]
 
+.Prerequisites
+* The prepared host must meet {Project} {InstallingServerDocURL}storage-requirements_{project-context}[Storage requirements].
+* The prepared host has base operating system repositories enabled.
+ifdef::orcharhino[]
+* The prepared host has the same content available as your {ProjectServer}.
+endif::[]
```

Comment:
I think this is no longer true. Previously it was needed for `postgresql-evr` but now base OS repos should suffice.
```suggestion
```

Reply:
The information is wrapped in an `orcharhino` check as it was on the old page -- @maximiliankolb does this need to stay for some orcharhino-specific reason?

Reply:
Yes, Ewoud's suggestion is correct because external DBs no longer need packages from Foreman/Katello.